### DISCUSSION TITLE
smplayer no longer works on new PC

### DISCUSSION THREAD

**Discussion by kurtigoedel** [smplayer-dev/smplayer#1106] [Q&A] [answered]
I have ported my Linux system (MX-Linux 23.4, Debian-based) from a Minisforum UM780XTX (AMD Ryzen 7 7840HS) to a Minisforum AI 370 (AMD Ryzen AI 9 HX 370).
Simply removed the SSD and installed it in the new mini PC. 

Almost everything continued to work, unfortunately not the smplayer.
When I now open a video with the smplayer, the corresponding mpv-player opens in a small window in the background (!!) and I have to manually bring this window to the foreground and enlarge it (full screen).
I have experimented with the smplayer settings, but so far without any success.

A heartfelt request to those in the know: What can I do?
Thanks in advance for any help.

**Comment by smplayer-dev** [comment]
Try selecting another video output in preferences -> general -> video.

**Reply by kurtigoedel** [reply, marked as answer]
Thank you very much, this worked.
I changed it from "standard" to "x11"
